projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.ts

```diff
@@ -154,10 +156,25 @@ export class TaskDetailsPageComponent implements OnInit {
         if(data.isChecked && data.upload){
           this.routingService.navigate(`/add-files/${this.projectDetails._id}`,{taskId:this.taskId})
         }else{
-          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG')
+          this.toasterService.showToast('ACCEPT_POLICY_ERROR_MSG',"danger")
         }
       }
     })
   }
 
+  onDateChange(newDate: Date) {
+    let localDateString = this.formatDateToLocal(newDate);
```

Comment:
Directly use
new Date(newDate).toISOString()

Reply:
the function is returning the one day previous date while rendering the date in the task list

Reply:
After selecting the date and then clicking on the datePicker again, is the date which u previously selected highlighted or is it changed?